src/Components/test/E2ETest/Tests/WebAssemblyStringComparisonTest.cs

```diff
@@ -21,7 +21,7 @@ public class WebAssemblyStringComparisonTest : ServerTestBase<ToggleExecutionMod
         {
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/runtime/issues/38126")]
```

Comment:
Would also be good to have an issue against Blazor for this to be re-enabled, unless it's tracked some other way.